ssas/service/admin/api.go

```diff
@@ -336,57 +339,69 @@ Responses:
 */
 func createSystem(w http.ResponseWriter, r *http.Request) {
 	sys := ssas.SystemInput{}
+	ssas.SetCtxEntry(r, "Op", "CreateSystem")
+	logger := ssas.GetCtxLogger(r.Context())
 	if err := json.NewDecoder(r.Body).Decode(&sys); err != nil {
-		service.JSONError(w, http.StatusBadRequest, "invalid request body", "")
+		logger.Error()
+		service.JSONError(w, http.StatusBadRequest, http.StatusText(http.StatusBadRequest), "")
 		return
 	}
 
-	ssas.OperationCalled(ssas.Event{Op: "RegisterClient", TrackingID: sys.TrackingID, Help: "calling from admin.createSystem()"})
+	logger.Infof("Operation Called: admin.createSystem()")
 	creds, err := ssas.RegisterSystem(r.Context(), sys.ClientName, sys.GroupID, sys.Scope, sys.PublicKey, sys.IPs, sys.TrackingID)
 	if err != nil {
-		service.JSONError(w, http.StatusBadRequest, fmt.Sprintf("could not create system; %s", err), "")
+		logger.Errorf("failed to create system; %s", err)
```

Comment:
can we add a comment in here to come back to the error returned at a future time? It looks like this may not necessarily be a bad request, it may be a more temporary status.

Reply:
Maybe there needs to be a ticket looking at the errors we return in SSAS overall; outside of the scope of this work, though

Reply:
yup, I actually have a placeholder ticket for this to address the return statuses because many of them are not helpful or do not fit the outcome of the error. https://jira.cms.gov/browse/BCDA-7954

I have also created another ticket to update some of our errors that we are returning from functions that make database calls, since those may be inaccurate: https://jira.cms.gov/browse/BCDA-7955